src/shared/components/query/QueryStore.ts

```diff
@@ -143,46 +145,35 @@ export class QueryStore
 	}
 
 	@observable studiesHaveChangedSinceInitialization:boolean = false;
-	@observable savedVirtualCohorts:VirtualCohort[] = [];
+	@observable userVirtualStudies:VirtualStudy[] = [];
+	@observable deletedVirtualStudies:string[] = [];
 
 	@action public deleteVirtualCohort(id:string) {
-		VirtualCohorts.delete(id);
+		sessionServiceClient.deleteVirtualStudy(id)
+		.then(() => {
+			this.deletedVirtualStudies.push(id)

```

Comment:
we probaby don't need to bother waiting for request to respond remove it from deletedVirtualStudies.  If there's a service error, then we show an error message, otherwise, assume success?